projects/lib/src/c4dt.css

```diff
@@ -6,19 +6,33 @@
   --blue: #0091aa;
   --light-blue: #a4bfcf;
 }
-.body {
+body,
+.mat-typography,
+.mat-body,
+.mat-body-1,
+.mat-select,
+.mat-select-value,
+.mat-tab-label,
+.mat-tab-link,
+.mat-form-field,
+.mat-stepper-vertical,
+.mat-form-field-appearance-legacy .mat-form-field-label,
+.mat-card,
+.mat-select-disabled {
```

Comment:
It seems that there is an easier way to do it, did you try it?
https://stackoverflow.com/questions/43747518/angular-material-and-changing-fonts